firmware/battery.cpp

```diff
@@ -36,42 +36,24 @@ int readVBAT(void) {
 };
 
 uint8_t mvToPercent(float mvolts) {
-    uint8_t battery_level;
+  if(mvolts<3300)
+    return 0;
 
-    if (mvolts >= 3000)
-    {
-        battery_level = 100;
-    }
-    else if (mvolts > 2900)
-    {
-        battery_level = 100 - ((3000 - mvolts) * 58) / 100;
-    }
-    else if (mvolts > 2740)
-    {
-        battery_level = 42 - ((2900 - mvolts) * 24) / 160;
-    }
-    else if (mvolts > 2440)
-    {
-        battery_level = 18 - ((2740 - mvolts) * 12) / 300;
-    }
-    else if (mvolts > 2100)
-    {
-        battery_level = 6 - ((2440 - mvolts) * 6) / 340;
-    }
-    else
-    {
-        battery_level = 0;
-    }
+  if(mvolts <3600) {
+    mvolts -= 3300;
+    return mvolts/30;
+  }
 
-    return battery_level;
+  mvolts -= 3600;
+  return 10 + (mvolts * 0.15F );  // thats mvolts /6.66666666
```

Comment:
4200-3600=600/6.666=90 (+10) = 100%. That's ok....
3601-3600=0=0(+10) = 10% ? that's low